java-extensions/hive-reader/pom.xml

```diff
@@ -0,0 +1,196 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.starrocks</groupId>
+        <artifactId>java-extensions</artifactId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>hive-reader</artifactId>
+    <packaging>jar</packaging>
+
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <slf4j.version>1.7.36</slf4j.version>
+        <guava.version>12.0.1</guava.version>
+        <java-extensions.home>${basedir}/../</java-extensions.home>
+        <hadoop.version>3.3.6</hadoop.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.avro</groupId>
+            <artifactId>avro</artifactId>
+            <version>1.11.2</version>
+            <exclusions>
+                <exclusion>
```

Comment:
Is necessary to exclude?

You can specify log4j / slf4j related version explicitly.

Maybe you can refers to hudi-reader/pom.xml

The point here is to make sure logger uses log4j2.properties, not log4j.properties.

Because right now we put logging configuration in log4j2.properties.